app/src/main/res/values/strings.xml

```diff
@@ -576,4 +576,5 @@
         <item quantity="one">%d item selected</item>
         <item quantity="other">%d items selected</item>
     </plurals>
+    <string name="importer_help_ente_auth">Supply an Ente Auth export file. Currently only unencrypted files are supported.</string>
```

Comment:
Please move this string to spot where the other ``importer_help_*`` strings are grouped together.

Reply:
Fine :) I made the changes as you suggested.